src/helpers/templateConstructor.js

```diff
@@ -0,0 +1,23 @@
+const reactImportTemplates = require('../templates/react-imports');
+const styleImportTemplates = require('../templates/style-imports');
+const renderContentTemplates = require('../templates/render-contents');
+const componentDefinitionTemplates = require('../templates/component-definition');
+const belowComponentTemplates = require('../templates/below-component');
+const exportsTemplates = require('../templates/exports');
+
+module.exports = (componentName, componentType, style) => {
+  const renderContents = renderContentTemplates(style);
+  const template = `
+  ${reactImportTemplates(componentType)}
+  ${styleImportTemplates(style, componentName)}
+
+  ${componentDefinitionTemplates(componentType, componentName, renderContents)}
+
+  ${belowComponentTemplates(style)}
+
+  ${exportsTemplates(style, componentName)}
+  `;
```

Comment:
Nit: So with the templates (even the ones I wrote), we don't indent the template string contents:

```js
const template = `
stuff
`;
```

Initially this was because I didn't want the exported file having a bunch of leading whitespace... but now that I think about it, I think prettier would trim all that, so it wouldn't be a problem.

In this case though, because it's already indented by the module.exports function, I think it's fine to indent:

```js
module.exports = (componentName, componentType, style) => {
  const template = `
    stuff
  `;
}
```

No change necessary for the other template strings though (unless you think it'd be better; I'm happy either way with the stuff in `templates/component-definition.js`)